### ISSUE TITLE
Upload a frontend project

### ISSUE THREAD

**Issue by gourav221b** [dsc-tiu/WebDev2022#1]
you can create some projects and upload them here

**Comment by Satyakoidala** [comment]
Hi @gourav221b ,

Can I provide a react js project to demonstrate auto suggestions implementation?

Thanks!

**Comment by RohitRajvaidya5** [comment]
Can I add project with only HTML, CSS and JavaScript?

**Comment by Ripan-Roy** [comment]
Hi @gourav221b 
I have made a tic-tac-toe game using react.js
Can I add it?

**Comment by tauquirahmed** [comment]
Hello @gourav221b 
I have made a simple shopping web page using HTML and CSS
Can I please add it?

**Comment by gourav221b** [comment]
> Hello @gourav221b I have made a simple shopping web page using HTML and CSS Can I please add it?

I'm creating a new branch for you. you can add it there.

**Comment by gourav221b** [comment]
> Hi @gourav221b I have made a tic-tac-toe game using react.js Can I add it?

@Ripan-Roy  I'm creating a new branch for you . you can add it there

**Comment by gourav221b** [comment]
> Hi @gourav221b ,
> 
> Can I provide a react js project to demonstrate auto suggestions implementation?
> 
> Thanks!

@Satyakoidala  Yes sure. I'm creating a new branch for you. kindly create a PR on that
